store/store.go

```diff
@@ -337,6 +344,12 @@ type searchResults struct {
 	} `json:"_embedded"`
 }
 
+type sectionResults struct {
+	Payload struct {
+		Sections []struct{ Name string } `json:"clickindex:sections"`
+	} `json:"_embedded"`
```

Comment:
I generally don't complain much about store APIs since they are invisible to the end user, but is there much benefit in holding up to those unfriendly names?